src/CommandLine/Text/HelpText.cs

```diff
@@ -227,22 +226,25 @@ public SentenceBuilder SentenceBuilder
         /// <param name='onError'>A delegate used to customize the text block of reporting parsing errors text block.</param>
         /// <param name='onExample'>A delegate used to customize <see cref="CommandLine.Text.Example"/> model used to render text block of usage examples.</param>
         /// <param name="verbsIndex">If true the output style is consistent with verb commands (no dashes), otherwise it outputs options.</param>
-        /// <param name="maxDisplayWidth">The maximum width of the display.</param>
-        /// <remarks>The parameter <paramref name="verbsIndex"/> is not ontly a metter of formatting, it controls whether to handle verbs or options.</remarks>
+        /// <param name="config">The configuration for the help-text</param>
+        /// <remarks>The parameter <paramref name="verbsIndex"/> is not only a metter of formatting, it controls whether to handle verbs or options.</remarks>
+        //REVIEW - I believe this is only public for test purposes and is not a client API hence I have not supplied 
+        //a backwards compatibility version

```

Comment:
After reading some of the old issues, it does look like calling this was the recommended way to get custom output so a backwards compatibility version is probably needed unless the major version is bumped.